frontend/src/redux/effects/index.js

```diff
@@ -768,7 +768,7 @@ function* saveDebt() {
 
   let body = {
     client: id,
-    debt_monthly_amount: loanDebt,
+    debt_monthly_amount: Number(loanDebt),
```

Comment:
i think you'll want to do Math.ceil(Number(loanDebt)) so that it wont ever show up as a decimal